src/js/app.js

```diff
@@ -1,5 +1,103 @@
 import $ from "jquery";
+import "slick-carousel";
 
-$(function() {
-  console.log('環境構築完了');
-})
+$(function () {
+  // Smooth Scroll
+  $(function () {
+    $('a[href^="#"]').on('click', function () {
+      var href = $(this).attr("href");
+      var target = $(href == "#" || href == "" ? 'html' : href);
+      var position = target.offset().top - $(".l-header").height();
+      $('body,html').animate({ scrollTop: position }, 600, 'swing');
+      $(".p-hamburgermenu").toggleClass("u-active__menu");
+      $(".p-hamburgermenu__icon").toggleClass("u-active__close");
+      noScroll();
+      return false;
+    });
+  });
+
+  // slide menu
+  $(function() {
+    $(".p-hamburgermenu__icon").on("click", function() {
+      $(".p-hamburgermenu").toggleClass("u-active__menu");
+      $(".p-hamburgermenu__icon").toggleClass("u-active__close");
+      $(".p-hamburgermenu").addClass("u-active__transitionTime");
+      noScroll();
+    });
+  });
+
+  // no scroll
+  function noScroll() {
+    if ($(".u-active__menu").length) {
+      $("body").css('overflow', 'hidden');
+    } else {
+      $("body").css('overflow', 'auto');
+    }
+  }
+
+  // slick
+  const breakPointWidth = 768;
+  let windowWidth = window.innerWidth;
+  const slideNum = $(".p-information__list").length;
+
+  function slickPC() {
+    $(".js-slider").not('.slick-initialized').slick({
+      infinite: true,
+      arrow: true,
+      slidesToShow: 4,
+      slidesToScroll: 4,
+    });
+  }
+
+  function slickSP() {
+    $(".js-slider").not('.slick-initialized').slick({
+      infinite: true,
+      arrow: true,
+      slidesToShow: 1,
+      slidesToScroll: 1
+    });
+  }
+
+  function slickConfirm() {
+    if (windowWidth >= breakPointWidth) {
+      if ($('.slick-initialized').length > 0) {
+        if (slideNum < 4) {
+          $('.slick-initialized').slick('unslick');
+        } else {
+          $('.slick-initialized').slick('unslick');
+          slickPC();
+        }
```

Comment:
[imo]
`$('.slick-initialized').slick('unslick');`はどちらの条件の時も実行すると思うので、else減らせるかなと思いました！

```suggestion
      if ($('.slick-initialized').length > 0) {
        $('.slick-initialized').slick('unslick');
        if (4 < slideNum) {
          slickPC();
        }
```

Reply:
ありがとうございます！
悩んだ末にif文を分けてしまったのですが、重複していた箇所を省略いたしました。[98688db](https://github.com/gizumo-education/front-coding/pull/14/commits/98688db585696be64b86a1ff42afe46129c86ee8)